<ui version="4.0" >
 <class>lp_ladspa_slider_int</class>
 <widget class="QWidget" name="lp_ladspa_slider_int" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>87</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <widget class="QSlider" name="sl_val" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QSpinBox" name="sp_val" />
   </item>
   <item row="1" column="0" >
    <widget class="QPushButton" name="pb_def_val" >
     <property name="maximumSize" >
      <size>
       <width>30</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="txt_name" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
